packages/horizontal-slider-vdom/src/types/FrameworkProps.ts

```diff
@@ -0,0 +1,11 @@
+type Ref<TElement> = {
+  current: TElement;
+};
```

Comment:
I'm wondering how that will play out with Vue because [their refs are strings](https://v3.vuejs.org/guide/component-template-refs.html).

Reply:
Since we will be in control of the Vue implementation, we could define a fallback type for it